src/components/MapView/ahmedabad.svg

```diff
@@ -0,0 +1 @@
1 1.23h-.37l-.01-1.23h.37zm-.16-1.5l.55.61-.21.39h-.75l-.18-.36.59-.64zm-.65 7.8h1.27v.36h-1.27v-.36zm1.27-.5h-1.27v-3.98h1.27v3.98zm-11.7-5.8l.01 1.23h-.37l-.01-1.23h.37zm-.16-1.5l.55.61-.21.39h-.75l-.18-.36.59-.64zm-.65 7.8h1.27v.36h-1.27v-.36zm1.27-.5h-1.27v-3.98h1.27v3.98zm-3.35 6.43c.23.22.23.53.23.55l.01.59h-.43l-.01-.6a.83.83 0 0 1 .2-.54zm-6.09 7.96c0-.02-.1-1.89.75-2.8.29-.32.66-.47 1.12-.48.37.01.7.12.96.39.84.86.78 2.82.78 2.85l-.02 4.02H5.61l-.03-3.98zm7.71 1.42l2.2.02v2.45h-2.2v-2.47zm2.2-.48l-2.2-.02V25.2h2.2v7.18zm.5-7.19h8.04v10.22h-.86l.04-7.19c0-1.56-1.42-2.84-3.18-2.84-1.75 0-3.18 1.27-3.18 2.84v7.19h-.87V25.19zm-2.97-4.09l.16-.42h2.42l.16.42h-2.74zm2.47-2.68v1.75h-2.2v-1.75h2.2zm.5 5.52h8.04v.75h-8.04v-.75zm.89-1.02c0-.2.12-.31.2-.37.23.15.23.33.23.37l.01.38h-.43l-.01-.38zm2.73-.04c0-.2.12-.31.2-.37.23.15.23.33.23.37l.01.38h-.43l-.01-.38zm2.68.02c0-.2.12-.31.2-.37.22.15.23.33.23.37l.01.38h-.43l-.01-.38zm2.24 12.42v-2.46l2.2.02v2.44h-2.2zm0-2.96v-7.17h2.2v7.19l-2.2-.02zm6.29-.92c0-.02-.1-1.89.75-2.8.29-.32.66-.47 1.12-.48.39-.02.7.12.96.39.84.86.78 2.82.78 2.85l-.02 4.02h-3.56l-.03-3.98zm-8.15 3.97h-5.31v-7.19c0-1.29 1.2-2.34 2.68-2.34s2.68 1.05 2.68 2.34l-.05 7.19zM37.3 23.48c.23.22.23.53.23.55l.01.59h-.43l-.01-.6c0-.28.11-.45.2-.54zm-7.45.49a3 3 0 0 1 6 0c0 .24-.04.49-.1.72h-.71l-.01-.66c0-.03.02-.7-.59-1.08l-.11-.07-.12.05c-.02.01-.6.26-.61 1.09l.02.66h-1.57l-.01-.66c0-.03.02-.7-.59-1.08l-.11-.07-.12.05c-.02.01-.6.26-.61 1.09l.02.66h-.67a2.28 2.28 0 0 1-.11-.7zm4.25.06a.8.8 0 0 1 .2-.55c.23.22.23.53.23.55l.01.59h-.43l-.01-.59zm-3 0a.8.8 0 0 1 .2-.55c.23.22.23.53.23.55l.01.59h-.43l-.01-.59zm6.85 1.16c.03 0 .05.02.05.05v10.12c0 .03-.02.05-.05.05h-3.04l.02-4.01c0-.09.07-2.19-.92-3.21a1.8 1.8 0 0 0-1.33-.54c-.6.01-1.09.22-1.48.64-.99 1.07-.89 3.08-.88 3.16l.03 3.96h-3.12v-2.52h.33v-.5h-.33v-7.2h10.72zm-11.22-.5h-2.2V21.6h2.2v3.09zm0-6.27v1.75h-2.2v-1.75h2.2zm-2.25 2.26h2.37l.16.42h-2.72l.19-.42zm-4.5-.67c1.66 0 3.21.89 4.05 2.32v1.1h-.79l-.01-.53c.01-.22-.11-.64-.62-.88l-.09-.04-.09.03a.92.92 0 0 0-.63.89l.02.53h-1.25l-.02-.55c.01-.22-.11-.64-.62-.88l-.09-.04-.09.04a.92.92 0 0 0-.63.89l.02.55h-1.3l-.01-.51c.01-.22-.11-.64-.62-.88l-.09-.04-.09.03a.92.92 0 0 0-.63.89l.02.52h-.41v-1.21c.14-.23.31-.44.48-.63h.02l-.01-.01a4.53 4.53 0 0 1 3.48-1.59zm-6.69 1.59h2.2v3.09h-2.2V21.6zm-.5 10.75h-.3v.5h.3v2.56H9.67l.03-4.01c0-.09.07-2.19-.92-3.21a1.72 1.72 0 0 0-1.33-.54c-.6.01-1.09.22-1.48.64-.99 1.07-.89 3.08-.89 3.16l.03 3.96H2.1c-.03 0-.05-.02-.05-.05V25.24c0-.03.02-.05.05-.05h10.69v7.16zm-7.32-8.32c0-.29.11-.46.2-.55.23.22.23.53.23.55l.01.59h-.42l-.02-.59zm3 0c0-.29.11-.46.2-.55.23.22.23.53.23.55l.01.59h-.42l-.02-.59zm-4.29-.06a3 3 0 0 1 6 0c0 .25-.04.49-.1.72h-.66l-.01-.66c0-.03.02-.7-.59-1.08l-.12-.06-.12.05c-.02.01-.6.26-.61 1.09l.02.66H6.42l-.01-.66c0-.03.02-.7-.59-1.08l-.12-.06-.12.05c-.02.01-.6.26-.61 1.09l.02.66h-.72a2.84 2.84 0 0 1-.09-.72zm-1.51-.49c.23.22.23.53.23.55l.01.59h-.42l-.01-.6c0-.28.11-.45.19-.54z"></path></g></svg>
```

Comment:
remove this file

Reply:
Using it as default image for the smaller icon, bro. Vo chhote wale me original image achhi nahi lag rahi isliye.

Fir bhi tu bole to kar deta hun

Reply:
> Using it as default image for the smaller icon, bro. Vo chhote wale me original image achhi nahi lag rahi isliye.
> 
> Fir bhi tu bole to kar deta hun

Smaller icon matlab smaller image in the Map card

Reply:
no you are not, see the code `import defaultImage from '../../assets/location/ahmedabad.svg';`